share_extension/icons/channel_type.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Any thoughts on if we'll need these icons in the main app? I guess we could move them if that ends up being the case